cdisutils/excel.py

```diff
@@ -132,7 +132,7 @@ def read_sheet(ws=None, num_headers=1):
                 break
 
         if any_data:
-            line_data = dict(zip(header, [str(cell.value).strip() for cell in row]))
+            line_data = dict(list(zip(header, [str(cell.value).strip() for cell in row])))
```

Comment:
This another case where the `list()` shouldn't be needed. `dict(zip(...))` should be fine in both Python versions.